config/crd/bases/operators.operatorframework.io_operators.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     listKind: OperatorList
     plural: operators
     singular: operator
-  scope: Namespaced
+  scope: Cluster
```

Comment:
Oh sorry, I submitted #82 for this. Should we merge that to keep separation of concerns and then rebase this one?

Reply:
Yes, no problem! I'll back out that change here then rebase after yours merges.